paseos/tests/actor_builder_test.py

```diff
@@ -95,3 +97,39 @@ def test_add_comm_device():
     assert len(sat1.communication_devices) == 2
     assert sat1.communication_devices["dev1"].bandwidth_in_kbps == 10
     assert sat1.communication_devices["dev2"].bandwidth_in_kbps == 42
+
+
+def test_set_geometric_model():
+    """Check if we can set the geometry, and if the moments of inertia are calculated correctly"""
+    _, sat1, _ = get_default_instance()
+    ActorBuilder.set_cuboid_geometric_model(sat1, mass=100, height=0.5, length=0.5, width=0.5)
+
+    assert sat1.mass == 100
+    assert round(sat1._moi[0,0],3) == 4.167      # Value for a h,w,l of each 0.5
+    assert sat1._moi[0,1] == 0.0                   # Should be zero if the mass distribution is even
+
+def test_view_geometric_model():
+    """Provide a visualisation of the imported geometric model, and test if the cg is in the correct location
+    """
+    _, sat1, _ = get_default_instance()
+    ActorBuilder.set_geometric_model_from_import(sat1,mass=100,file_name='20265_Hexagonal_prism_v1') # test only runs
+    # if '../../' is added to start of the file name in geometric_model.pu
+    assert np.all(np.round(sat1._center_of_gravity, decimals=0) == np.array([0, 0, 6]))
+    # Create trace for the mesh and axis
+    mesh = sat1._mesh
+    mesh_trace = go.Mesh3d(x=mesh.vertices[:, 0], y=mesh.vertices[:, 1], z=mesh.vertices[:, 2],
+                           i=mesh.faces[:, 0], j=mesh.faces[:, 1], k=mesh.faces[:, 2],
+                           color='lightblue', opacity=0.5)
+    axis_length = 5  # Length of the axes
+    axis_end = [[0, 0, 0], [axis_length, 0, 0]], [[0, 0, 0], [0, axis_length, 0]], [[0, 0, 0], [0, 0, axis_length]]
+    axis_traces = [go.Scatter3d(x=[end[0][0], end[1][0]], y=[end[0][1], end[1][1]], z=[end[0][2], end[1][2]],
+                                marker=dict(color=col), line=dict(color=col, width=4), name=lab)
+                   for end, col, lab in zip(axis_end, ['red', 'green', 'blue'], ['X-axis', 'Y-axis', 'Z-axis'])]
+    point_coordinates = sat1._center_of_gravity
+
+    # Create a trace for the single point
+    point_trace = go.Scatter3d(x=[point_coordinates[0]], y=[point_coordinates[1]], z=[point_coordinates[2]],
+                               mode='markers', marker=dict(size=10, color='orange'), name='Center of Gravity')
+    fig = go.Figure(data=[mesh_trace] + axis_traces + [point_trace])
+    fig.update_layout(scene=dict(aspectmode='data'))
+    fig.show()
```

Comment:
Would avoid this test / tests with plotting. We run the tests automatically on github without UI

Reply:
I'll remove it